go-controller/pkg/network-controller-manager/node_network_controller_manager.go

```diff
@@ -150,35 +127,32 @@ func (ncm *nodeNetworkControllerManager) Run(ctx context.Context) error {
 		}, time.Minute, ncm.stopChan)
 	}
 
+	ncm.defaultNodeNetworkController = node.NewDefaultNodeNetworkController(ncm.newCommonNetworkControllerInfo())
 	err = ncm.defaultNodeNetworkController.Start(ctx)
 	if err != nil {
 		return fmt.Errorf("failed to start default network controller: %v", err)
 	}
 
 	// nadController is nil if multi-network is disabled
 	if ncm.nadController != nil {
-		klog.Infof("Starts net-attach-def controller")
-		return ncm.nadController.Run(ncm.stopChan)
+		return ncm.nadController.Start()
```

Comment:
Do we need to stop already running processes like `defaultNodeNetworkController` or `checkForStaleOVSInternalPorts` in case error happens?

Reply:
ups I didn't notice I removed the defer.

Reply:
Added back